lib/apollo-client.ts

```diff
@@ -147,8 +148,16 @@ function createLink({ twoFactorAuthContext }) {
 
   const linkFetch = process.browser ? fetch : serverSideFetch;
 
-  const apiV1DefaultLink = new HttpLink({ uri: getGraphqlUrl('v1'), fetch: linkFetch });
-  const apiV2DefaultLink = new HttpLink({ uri: getGraphqlUrl('v2'), fetch: linkFetch });
+  const apiV1DefaultLink = createUploadLink({
+    uri: getGraphqlUrl('v1'),
+    fetch: linkFetch,
+    headers: { 'Apollo-Require-Preflight': 'true' },
```

Comment:
It's great that we're doing this, but I think we should extract everything related to CSRF into separate PRs and make sure we deploy them in the right order, with a grace period to not break anything for users that will potentially visit the website from a tab that is a few days old.